src/elmish/App.fs

```diff
@@ -0,0 +1,232 @@
+module Pixi
+
+open Fable.Core
+open Fable.Core.JsInterop
+open Fable.Import
+open Fable.Import.Pixi
+open Fable.Import.Browser
+open Fable.Import.JS
+open Fable.Pixi
+open Elmish
+open Elmish.React
+open Fulma.Components
+open Fulma.Elements
+open Fulma.Layouts
+open Fable.Helpers.React
+open Fable.Helpers.React.Props
+
+// Fulma css
+importAll "../../public/sass/main.sass"
```

Comment:
I prefer to use `importSideEffect` in general when importing file to create a side effect in webpack. (for info importSideEffect is "new")

Reply:
Done 👍